### PR TITLE
Fixed logging of fails.

### LINKED ISSUE #7: Delete package-lock.json when running `skyux install`

### PR DESCRIPTION
**Pull request by Blackbaud-BobbyEarl**
Fixes #7, #9, and #10.

### CODE REVIEW

File: lib/npm-install.js [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
+const logger = require('@blackbaud/skyux-logger');
 
 /**
  * Runs npm install for a specific package
  * @name npmInstall
  */
 function npmInstall(settings) {
-  const message = logger.promise('Running npm install (can take several minutes)');
 
+  let messageText = 'Running npm install. This step can take several minutes.';
```
Comment (Blackbaud-SteveBrush) [inline lib/npm-install.js]: In other console logs, we're wrapping the command in back ticks. Curious if we need to make it consistent?

File: lib/npm-install.js [hunk 2]
```diff
@@ -19,19 +18,30 @@ function npmInstall(settings) {
...
+    }
   }
 
+  const message = logger.promise(messageText);
   const npmProcess = spawn('npm', ['install'], installArgs);
 
   return new Promise((resolve, reject) => {
+    let error = '';
+
+    // This doesn't exist if we're logging verbosely
```
Comment (Blackbaud-SteveBrush) [inline lib/npm-install.js]: By "this", do you mean the `stderr` property of the process?

File: lib/npm-install.js (hunk 1 above)
Comment (Blackbaud-BobbyEarl) [reply lib/npm-install.js]: I can.  Within this file; however, none of strings are wrapped in backticks.  I've typically followed the pattern of only using backticks if it needs to be a string literal which has an expression to be evaluated.

File: lib/npm-install.js (hunk 2 above)
Comment (Blackbaud-BobbyEarl) [reply lib/npm-install.js]: Yes.  I'll clarify the comment.

File: lib/npm-install.js [hunk 3]
```diff
@@ -30,7 +30,7 @@ function npmInstall(settings) {
   return new Promise((resolve, reject) => {
     let error = '';
 
-    // This doesn't exist if we're logging verbosely
+    // stderr property doesn't exist is spawned with stdio === 'inherit'
```
Comment (Blackbaud-SteveBrush) [inline lib/npm-install.js]: Should the "is" be "if"?

Comment (Blackbaud-BobbyEarl) [reply lib/npm-install.js]: Let me have my typos and be gone.  JK, fixed it.  :-)

**Pull request merged**